projects/plugins/jetpack/extensions/blocks/subscriber-login/subscriber-login.php

```diff
@@ -129,7 +129,7 @@ function render_block( $attributes ) {
 		return sprintf(
 			$block_template,
 			get_block_wrapper_attributes(),
-			'https://wordpress.com/read/subscriptions',
+			'https://wordpress.com/read/subscriptions?s=' . get_site_url(),
```

Comment:
Let's encode the query param.

```suggestion
			'https://wordpress.com/read/subscriptions?s=' . rawurlencode( get_site_url() ),
```